README.md

```diff
@@ -77,13 +80,50 @@ Please see [our Showcase page](https://weareadaptive.com/showcase/) for a full l
    ```
    VITE_HYDRA_URL=ws://localhost:8929 npm start
    ```
-</details>
-
+   </details>
 
 ## CI/CD
 
 We practice continuous integration and deployment. Every branch and pull request triggers a build and deployment to an ephemeral environment. Merging to master causes a build and deployment to our [development environment](https://web.dev.reactivetrader.com).
 
+## Reactive Launcher
+
+<details open>
+<summary>Run Reactive Trader® Launcher locally</summary>
+
+Run:
+
+```
+npm run openfin:run:launcher
+```
+
+</details>
+
+Reactive Trader® Launcher interprets and executes tasks based on natural language commands expressed by the user.
+It serves as a convenient launchpad for a range of other Reactive applications, such as Reactive Trader FX (RTFX), Reactive Trader Credit (RTC), Reactive Analytics (RA), and Limit Checker (LC).
+
+The Reactive Trader® Launcher search engine supports a wide range of commands.
+Here are a few examples:
+
+```
+Buy USDJPY 10M
+How is Cable doing?
+How is the $currency market?
+How is the $currency doing?
+Let me know if a GBP trade is over 1 mil
+Let me know if a GBPUSD trade is over 1 mil
+Notify me about large trades
```

Comment:
really? how does it notify the user?
I always wondered what the full extent of the NLP was 😸

Reply:
I have tested it out and it seems to not show anything but in the google dialogflow it is defined as a training phrase - prob something to look into

Reply:
you can test whether it recognizes the intent (in your response code)
might just be that we do not pick up / do anything with that intent "type" or whatever we call each intent instance ...

Reply:
I dont think we have any parameters & actions set for notification. For now the only custom entities we have are currency pair, trade type and currency